crates/uv/Cargo.toml

```diff
@@ -136,33 +136,44 @@ ignored = [
 ]
 
 [features]
-default = ["python", "python-managed", "pypi", "git", "performance", "crates-io", "slow-tests", "test-ecosystem"]
+default = ["performance", "uv-distribution/static", "default-tests"]
 # Use better memory allocators, etc.
-performance = [
-    "performance-memory-allocator",
-    "uv-distribution/performance",
-]
+performance = ["performance-memory-allocator"]
 performance-memory-allocator = ["dep:uv-performance-memory-allocator"]
+
+# Adds self-update functionality. This feature is only enabled for uv's cargo-dist installer
+# and should be left unselected when building uv for package managers.
+self-update = ["axoupdater", "uv-cli/self-update"]
+
+# Features for development only.
 tracing-durations-export = ["dep:tracing-durations-export", "uv-resolver/tracing-durations-export"]
 
+# Features that only apply when running tests, no-ops otherwise.
+default-tests = [
+    "crates-io",
+    "git",
+    "pypi",
+    "python",
+    "python-managed",
+    "slow-tests",
+    "test-ecosystem"
+]
+# Introduces a dependency on crates.io.
```

Comment:
Can these be "test dependency"?

Reply:
Cargo has dev dependencies (and build dependencies), but they are always obligatory dependencies, while optional dependencies are always mandatory (https://github.com/rust-lang/cargo/issues/1596)

Reply:
or "test suite" or something

Reply:
ah lol